src/ui/title-ui-handler.ts

```diff
@@ -63,8 +63,8 @@ export default class TitleUiHandler extends OptionSelectUiHandler {
       .then(request => request.json())
       .then(stats => {
         this.playerCountLabel.setText(`${stats.playerCount} ${i18next.t("menu:playersOnline")}`);
-        if (this.splashMessage === battleCountSplashMessage)
-          this.splashMessageText.setText(battleCountSplashMessage.replace('{COUNT}', stats.battleCount.toLocaleString('en-US')));
+        if (this.splashMessage === getbattleCountSplashMessage())
```

Comment:
```suggestion
        if (this.splashMessage === getBattleCountSplashMessage())
```
Rename this for proper casing